app/ui/shared/widgets/overview/cards/overview-card.jsx

```diff
@@ -29,7 +29,8 @@ class OverviewCard extends Component {
   render() {
     const meta = this.props.page.meta;
 
-    if (meta.rating || meta.price) {
+    if (meta.price ||
+        meta.rating && parseFloat(meta.rating) > 0) {
```

Comment:
You're gonna want to wrap `meta.rating && parseFloat(meta.rating) > 0` in parens here to silence eslint.